packages/babel-generator/src/generators/typescript.ts

```diff
@@ -435,7 +435,7 @@ export function TSInterfaceDeclaration(
   this.space();
   this.print(id, node);
   this.print(typeParameters, node);
-  if (extendz) {
+  if (extendz?.length > 0) {
```

Comment:
Could you also fix the typo? 😂🙏

Also just a nit but please remove `> 0`, since the coercion 0->false is easier to understand than whatever is going on with `undefined > 0`.

Reply:
Ya, np, fixed

Reply:
Some version of ts enforce a number vs number comparison, so usually write it like this `(arr?.length ?? 0) > 0`, but it's even worse IMO 😅